src/index.html

```diff
@@ -11,7 +11,150 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+    method="post"
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    >
+      <fieldset class="form-group">
+        <legend>Personal information</legend>
+
+        <label class="form-field">Surname:
+          <input
```

Comment:
Move text to another line. This will improve readibility
```suggestion
        <label class="form-field">
          Surname:
          <input
```